CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 - Remove copyright year and sign from license and attributions
 
 ### Fixed
+- Fix homepage image links that were too large in Internet Explorer [styleguide-only]
```

Comment:
@maya I still think we need to be explicit and state IE9 - 11, as we have not tested other versions of IE outside of those and don't want to make any assumptions.